android/src/main/java/com/marigold/rnsdk/JsonConverter.kt

```diff
@@ -9,7 +9,7 @@ import org.json.JSONObject
 /**
  * Class for handling conversion between Native Maps and Arrays and JSON Objects and Arrays.
  */
-internal class JsonConverter {
+class JsonConverter {
```

Comment:
Is there a reasons this class is being made public? It should only be required internally.